src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -50,7 +50,12 @@ export default function (props: BloodPressureVisualizationProps) {
     const yInterval: ClosedInterval = { values: [_minDiastolic, _maxSystolic] };
     const axis: Axis = { yRange: yInterval, yIncrement: 50, xIncrement: (80 / 7) };
     const [bloodPressureData, setDataForGraph] = useState<Map<string, BloodPressureDumbbell> | undefined>(undefined);
-    const [datePagerStartDate, setStartOfWeek] = useState<Date>(startOfDay(getWeekStart(props.weekStartsOn ?? "Monday")));
+    const dateRangeContext = useContext(DateRangeContext);
+    let intervalStart = dateRangeContext?.intervalStart ?? startOfDay(getWeekStart(props.weekStartsOn ?? "Monday"));
+
+    if ( dateRangeContext && dateRangeContext?.intervalType !== "Week"){
+        return <div className="mdhui-blood-pressure-interval-not-supported" ref={props.innerRef}>Blood Pressure Visualization does not support month view at this time</div>;
```

Comment:
I wonder if this should be updated to match the condition it is checking for.  At the moment, "month" is the only other possible option, but perhaps some other option might be added in the future. 

`Blood Pressure Visualization only supports "week" view at this time.`